File: servicio-orquestador/src/main/java/co/edu/unicauca/orquestador/fachadaServices/DTORespuesta/RespuestaConsultaPazySalvoDeportesDTO.java
package co.edu.unicauca.orquestador.fachadaServices.DTORespuesta;

import java.time.LocalDate;

import lombok.Data;

@Data
public class RespuestaConsultaPazySalvoDeportesDTO {
    private int codigoEstudiante; 
    private LocalDate fechaPrestamo;
    private LocalDate fechaDevolucionEstimada; 
    private LocalDate fechaDevolucionReal; 
    private String implemento;
}

File: app-estudiante/src/app/modelo/RespuestaConsultaPazySalvoDeportesDTO.ts
export class RespuestaConsultaPazySalvoDeportesDTO {
  codigoEstudiante!: number;
  fechaPrestamo!: string;             // formato 'YYYY-MM-DD'
  fechaDevolucionEstimada!: string;   // formato 'YYYY-MM-DD'
  fechaDevolucionReal!: string;       // formato 'YYYY-MM-DD'
  implemento!: string;
}